flag_preflight.py

```diff
@@ -0,0 +1,65 @@
+"""
+Naive warp:
+  pico the svg
+  convert everything to cubic
+  split the cubics (we'll just assume too coarse initially)
+  warp start/end and intermediate points directly
+    no fancy tricks with derivatives... my notes said we should have some
+
+Note the lack of:
+  curve simplification
+  intelligent splitting (only when error introduced)
+
+Usage:
+  python naive_warp.py 1f1e6_1f1ec.svg
+  python naive_warp.py noto-emoji/third_party/region-flags/svg/GB-WLS.svg --out_file GB-WLS-waved.svg
+"""
+from absl import app
+from absl import flags
+
+import enum
+from helpers import *
+from lxml import etree
+from picosvg.svg_meta import ntos, splitns, svgns, xlinkns
+import re
+
+DEFAULT_PRECISION = 200
+SHOULD_RENAME = {
+    "store-width": "stroke-width",
```

Comment:
we should probably fix the sources

Reply:
flags are unusually hands-off; apparently waveflag.c was ok so I'm trying to follow the same pattern